### PR TITLE
add getter for serverHostname so classes based on this can access the…

### PR DESCRIPTION
**Pull request by dballard**
…ir address

For instance if I have a class 

type RicochetMobileService struct {
	goricochet.StandardRicochetService
}

it would be nice after loading to be able to determine what the hostname is to return and display (I know its logged but that's not a good UI)

### CODE REVIEW

File: standardricochetservice.go [hunk 1]
```diff
@@ -180,3 +180,9 @@ func (srs *StandardRicochetService) OnBadUsageError(oc *OpenConnection, channelI
 func (srs *StandardRicochetService) OnFailedError(oc *OpenConnection, channelID int32) {
 	oc.RejectOpenChannel(channelID, "FailedError")
 }
+
+// Getters
```
Comment (s-rah) [inline standardricochetservice.go]: Remove

File: standardricochetservice.go [hunk 2]
```diff
@@ -180,3 +180,9 @@ func (srs *StandardRicochetService) OnBadUsageError(oc *OpenConnection, channelI
...
+
+func (srs *StandardRicochetService) GetServerHostname() string {
```
Comment (s-rah) [inline standardricochetservice.go]: No godoc...tut tut

Comment (s-rah) [inline standardricochetservice.go]: Also, I wonder if this should also be part of the Interface...though I suppose an implementation could front multiple ricochet services....

Comment (s-rah) [inline standardricochetservice.go]: Or tests

**Comment by dballard** [comment]
Updated anyways for best practices. I'll keep extending where I need on this and other branches as I start to use it for a project. Might give you some ideas. thanks for the feedback! Feel free to close.
I'll probably make a master branch then with all my changes? or commit to my master? What would be easiest for you to keep an eye on?

**Comment by s-rah** [comment]
This this is covered by new_api now.

**Pull request closed without merging**